3rdparty/libtiff/tif_hash_set.c

```diff
@@ -367,7 +367,7 @@ static bool TIFFHashSetRehash(TIFFHashSet *set)
 {
     int nNewAllocatedSize = anPrimes[set->nIndiceAllocatedSize];
     TIFFList **newTabList =
-        (TIFFList **)(calloc(sizeof(TIFFList *), nNewAllocatedSize));
+        (TIFFList **)(calloc(nNewAllocatedSize, sizeof(TIFFList *)));
```

Comment:
Upstream of libtiff has fixed this: https://gitlab.com/libtiff/libtiff/-/blob/master/libtiff/tif_hash_set.c#L370